### ISSUE TITLE
Select addresses from a group when composing

### ISSUE THREAD

**Issue by rdacn** [roundcube/roundcubemail#7742]
Hi
we have a problem that might not be very common
Our users are essentially teachers
They import  pupils in their contacts using csv, from a student management software 
Every pupil has 2 mail addresses (mother and father, or parent1 and parent2), except in a few cases
Teachers then create groups (one class = one group)
But when they want to send a message to the whole group, they go to "insert contact(s)" they search for the group, they select the group and... only one address per pupil is retrieved (presumably the first one in the list)

The only solution I found is to tell them to edit each pupil contact entry and add a keyword (name of the class or group) to its first or last name, then to search in contacts with this keyword : this way, they can select all addresses

The real solution would not be to retrieve all addresses from every contact when selecting a group, as it would obviously lead to duplicates, but simply, when searching a group in 'insert contact(s)' , to display all addresses from this group and let the user select which ones (s)he wants to choose.

Thanks

**Comment by alecpl** [comment]
Most likely what should happen here is that 'group-expand' action should consider "Skip alternative email addresses in autocompletion" option. I.e. it should use all email addresses when `autocomplete_single=false` (default), but it would use one address when `autocomplete_single=true`.

So, it could be argued that this actually is a bug.

**Comment by rdacn** [comment]
That's true for auto-completion, but I'd like to point out that my primary target was not auto-completion but the menu that appears when you click on the "Add/insert contact(s)" button on the right.

In this menu, when you search for a group, you can only select the group itself. A better option would be to display the group, but also, each individual entry belonging to the group, with all its addresses, therefore providing a way to select the group by clicking it (as for now) OR select each individual address.

**Comment by clesmian** [comment]
We are facing a similar issue. After looking for a solution I stumbled upon this issue. 

I implemented a quick solution, which solves the problem of sending emails to multiple guardians of a child for us. I also created a PR. Maybe it also helps in your case.

If there are any comments, I'm always open to suggestions.

**Comment by Neustradamus** [comment]
@rdacn: Have you tested the @clesmian PR?

**Comment by gthieleb** [comment]
We are also facing the same issue. Teacher that saving two mail adresses in one contact can only select one. @rdacn where you able to test this PR?
